baselines/exo.md

```diff
@@ -796,55 +910,53 @@ At a minimum, the following alerts SHALL be enabled:
 
   - Suspicious Email Forwarding Activity.
 
-  - Unusual increase in email reported as phish.
-
   - Messages have been delayed.
 
   - Tenant restricted from sending unprovisioned email.
 
   - Tenant restricted from sending email.
 
-  - Malware campaign detected after delivery.
-
   - A potentially malicious URL click was detected.
+
```

Comment:
Right now we use bullet points for both the alerts and Rationale/Last modified. I would use a. b. c. d. to list the alerts shall be enabled to differentiate them